### ISSUE TITLE
version 0.9.8 breaks cssPath

### ISSUE THREAD

**Issue by gilbox** [aslansky/css-sprite#57]
The background-image URL in the generated file is incorrect with this release.

`/assets/blah/sprite.png` is transformed to `/assets/sprite.png`

**Comment by envex** [comment]
I found that I had to add a trailing slash to my cssPath option.

    cssPath: "../img/"

**Comment by douglasmiller** [comment]
The trailing slash inclusion is contrary to the documentation in the README:

```
gulp.task('sprites', function () {
  return gulp.src('./src/img/*.png')
    .pipe(sprite({
      name: 'sprite',
      style: '_sprite.scss',
      cssPath: './img',
      processor: 'scss'
    }))
    .pipe(gulpif('*.png', gulp.dest('./dist/img/'), gulp.dest('./dist/scss/')))
});
```

The [index.js](https://github.com/aslansky/css-sprite/blob/master/index.js#L29) also indicates that the default cssPath is set to `../images':

```
var defaults = {
  src: null,
  out: '',
  name: 'sprite',
  style: null,
  format: 'png',
  cssPath: '../images',
  processor: 'css',
  template: null,
  orientation: 'vertical',
  retina: false,
  background: '#FFFFFF',
  margin: 4,
  opacity: 0,
  sort: true
};

```

**Comment by mrpavlikov** [comment]
Had to add trailing slash aswell.

**Comment by message** [comment]
Actually it's 0.9.7 breaks cssPath.

**Comment by aslansky** [comment]
Issue should be fixed in [sprity v1.0.0](https://github.com/sprity/sprity)
